integration_tests/src/main/python/window_function_test.py

```diff
@@ -240,6 +246,36 @@ def test_decimal_running_sum_window_no_part(data_gen):
         'from window_agg_table',
         conf = {'spark.rapids.sql.batchSizeBytes': '100'})
 
+@ignore_order
+@approximate_float
+@pytest.mark.parametrize('data_gen', numeric_gens, ids=idfn)
+def test_numeric_running_sum_window_no_part_unbounded(data_gen):
```

Comment:
nit: Sorry I didn't catch this before, but this is not a running sum.  So ideally we just remove running from the name of the test.